kandroid/src/main/kotlin/com/pawegio/kandroid/KContext.kt

```diff
@@ -215,4 +217,13 @@ val Context.windowManager: WindowManager?
 val Context.defaultSharedPreferences: SharedPreferences
     get() = PreferenceManager.getDefaultSharedPreferences(this)
 
-inline fun <reified T : Any> Context.startActivity() = startActivity(IntentFor<T>(this))
\ No newline at end of file
+inline fun <reified T : Any> Context.startActivity() = startActivity(IntentFor<T>(this))
+
+fun Context.isPermissionGranted(permission:String):Boolean = ContextCompat.checkSelfPermission(this, permission) == PackageManager.PERMISSION_GRANTED
+fun Context.isPermissionsGranted(vararg permissions:String):Boolean {
+    permissions.forEach { permission ->
+        if(ContextCompat.checkSelfPermission(this, permission) == PackageManager.PERMISSION_DENIED)
+            return false
+    }
+    return true
```

Comment:
A little complicated, it can by simplified with following predicate:
`permissions.all { ContextCompat.checkSelfPermission(this, it) == PackageManager.PERMISSION_GRANTED }`